controllers/comments.controllers.js

```diff
@@ -0,0 +1,14 @@
+const { deleteCommentById } = require("../models/comments.models");
+
+const deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
Great stuff, res.sendStatus can also be used if not sending back a body (does the same thing but maybe a bit more elegant)